src/modules/fullCell.class.js

```diff
@@ -0,0 +1,41 @@
+export class fullCell {
```

Comment:
```suggestion
export class FullCell {
```
and filename in PascalCase